src/libcollections/str.rs

```diff
@@ -599,7 +599,7 @@ impl str {
     /// # #![feature(str_char, core)]
     /// use std::str::CharRange;
     ///
-    /// let s = "中华Việt Nam";
+    /// let s = "中华Việt Nam";
```

Comment:
Ah, yeah, that’s a bug. I thought you were saying there was not change at all there in the source.